wallet-new/test/unit/Wallet/Inductive/History.hs

```diff
@@ -0,0 +1,118 @@
+module Wallet.Inductive.History (
+    History -- opaque
+    -- * "Difference history" (like difference lists)
+  , HistoryD -- opaque
```

Comment:
Pure 🚴 shedding -- what about `DHistory` as in, literally, `DList`? 😁 

(Jokes aside -- this is testing code, so it doesn't really matter).

Reply:
To be honest, this is a relic of the old version of the `History`. I'll get rid of this type completely.